src/sentry/tasks/relocation.py

```diff
@@ -425,7 +423,7 @@ def cross_region_export_timeout_check(
     """
 
     try:
-        relocation: Relocation = Relocation.objects.get(uuid=uuid)
+        relocation = Relocation.objects.get(uuid=uuid)
```

Comment:
Just to be clear: `objects.get()` now returns a typed value as far as mypy is concerned? Or are we removing the case here for other reasons?

Reply:
from the date this code was written it always has -- this annotation was not needed